routes/v1.js

```diff
@@ -1236,10 +1271,11 @@ api.declare({
   name:       'reportCompleted',
   scopes: [
     [
+      // Legacy
       'queue:resolve-task',
       'assume:worker-id:<workerGroup>/<workerId>'
     ], [
-      'queue:claim-task:<taskId>/<runId>'
+      'queue:claim-run-id:<taskId>/<runId>'
```

Comment:
There is an assumption here that the same component that claims a task should report the state. In the case of an "adapter" worker, such as the fictitious "Jenkins Bridge", it may be that one (central) service claims tasks, but another connected service reports state and uploads artifacts. Better than `queue:claim-run-id:<taskId>/<runId>` here would be `queue:resolve-task:<taskId>/<runId>`. Initially, the Queue could provide this scope in temporary credentials to all workers that claim tasks, but later if we needed to restrict this, we would have a mechanism to enable us to do so (for example, purely for illustration; we might enforce a mechanism such that the fictitious Jenkins worker might not receive the `queue:resolve-task:<taskId>/<runId>` scope in the temporary credentials given to it when claiming a task, and the slaves might instead make a separate call to gain temporary credentials with this scope, by virtue of some other scope they may posses). This is just a way of future-proofing. As I say, to keep things as they currently are, Queue can give `queue:resolve-task:<taskId>/<runId>` in response to a task claim for now.